enclave-tls/samples/enclave-tls-server/server.c

```diff
@@ -151,7 +150,7 @@ int enclave_tls_server_startup(int sockfd, enclave_tls_log_level_t log_level,
 
 		printf("Client: %s\n", buf);
 
-#ifdef OCCLUM
+#if defined(OCCLUM) || defined(SGX)
 		sgx_report_t app_report;
```

Comment:
`sgx_create_report` is only for occlum.

Reply:
Same idea like above. sgx_report_t is unavailable for normal build.